genesis/process/disabled/feeHandler.go

```diff
@@ -22,7 +22,7 @@ func (fh *FeeHandler) MinGasPrice() uint64 {
 }
 
 // MaxGasLimitPerBlock return max uint64
```

Comment:
Reformulate this comment :)

Reply:
it really returns max uint64 as it is a disabled handler.